samples/nrf9160/memfault/overlay-modem-trace.conf

```diff
@@ -0,0 +1,37 @@
+# Modem trace
```

Comment:
Missing license

Reply:
Oh, this file wasn't supposed to be here. Will remove it.